tests/test_utils.py

```diff
@@ -395,3 +399,55 @@ def test_no_string(self):
 
         with self.assertRaisesRegex(TypeError, UNACCENT_TYPE_ERROR):
             unaccent_string(1234)
+
+
+class TestValidateField(TestCase):
+    """Unit tests for validate_field"""
+
+    def test_validate_string(self):
+        """Check valid and invalid fields"""
+
+        # If the field is valid, the method does not raise any exception
+        validate_field('test_field', 'Test')
+
+        validate_field('test_field', None, allow_none=True)
```

Comment:
This one is not needed because you are testing this parameter in a dedicated test.

Reply:
Ok.